blakserv/interface.h

```diff
@@ -43,4 +43,8 @@ void StartAsyncPortalSocket(SOCKET sock);
 
 void FatalErrorShow(const char *filename,int line,const char *str);
 
+#ifdef _WIN32

```

Comment:
Use BLAK_PLATFORM_WINDOWS instead.

Also I think it would be slightly better if the #ifdef were inside the body of ShowCopyableMessageDialog in interface.c instead.  That makes the #ifdef less visible than in a header file.